apps/meteor/client/startup/slashCommands.ts

```diff
@@ -11,7 +11,7 @@ Tracker.autorun(async () => {
 	if (oldUserId === null && newUserId) {
 		APIClient.get('/v1/commands.list').then((result) => {
 			result.commands.forEach((command) => {
-				slashCommands.add(command.command);
+				slashCommands.add(command.command, undefined, undefined, undefined, undefined, undefined, undefined, command.appId);
```

Comment:
Would it be possible to change the signature to accept an object instead of multiple parameters? This is looking weird, honestly 🙈

Reply:
Yeah totally agree with you, it would require a big change to refactor all files that use the `slashCommand.add`, but I think that is worth it.